moveos/moveos-commons/moveos-common/src/utils.rs

```diff
@@ -88,6 +88,6 @@ pub fn check_open_fds_limit(max_files: u64) -> Result<(), Error> {
 }
 
 #[cfg(not(unix))]
-pub fn check_open_fds_limit(_max_files: u64) -> Result<(), ConfigError> {
+pub fn check_open_fds_limit<ConfigError>(_max_files: u64) -> Result<(), ConfigError> {
```

Comment:
try function ?
```
#[cfg(not(unix))]
pub fn check_open_fds_limit(_max_files: u64) -> Result<(), Error>
```

Reply:
> try function ?
> 
> ```
> #[cfg(not(unix))]
> pub fn check_open_fds_limit(_max_files: u64) -> Result<(), Error>
> ```

Thank you very much, good suggestion, it is better to use Error for errors here